docker/filer.toml

```diff
@@ -0,0 +1,3 @@
+[memory]
+enabled = true
```

Comment:
how about default to leveldb? 

If just in memory, the directories will be gone while the files are still stored on volume servers.

Reply:
Great point, I assumed leveldb required some sort of extra setup, but just tested and it does preserve the information on a container restart. Although state will be lost if the container is rebuilt without the use of a volume of course.